packages/vapor/scss/components/selected-option.scss

```diff
@@ -12,7 +12,7 @@
     margin: $dropdown-multiselect-margin;
     color: var(--title-text-color);
     font-size: $button-small-font-size;
-    line-height: $dropdown-multiselect-option-height;
+    line-height: $dropdown-multiselect-option-height - 1px;
```

Comment:
Why not just put `23px` on the variable `$dropdown-multiselect-option-height`, so that we can have the `min-height` and the `line-height` equal?

By the way, what's the point of `min-height` if we still have text inside?

Reply:
I would tend to agree to just change the dropdown-multiselect-option-height to 23px directly if it doesnt break anything at some places else

Reply:
if you want passed `min-height` at `26px`, this to broke

Reply:
`min-height` makes sure the dividing line is the full height of the box. I made the line-height 1px smaller because I could see a tiny tiny difference (it sat too high) but the difference really is negligible